README.md

```diff
@@ -206,6 +206,39 @@ int main(void){
 }
 ```
 
+## Run-time trace group filtering
+
+You can control run-time the tracing per trace group. Each of your modules should should define the trace group (using `#define TRACE_GROUP "abcd"` in the .c/.cpp file) and you can use this trace group for controlling the tracing.
+
+| Function                          | Explanation                                                |
+|-----------------------------------|------------------------------------------------------------|
+|`mbed_trace_include_filters_get()` | Get pointer to the component exclusion filter list.        |
+|`mbed_trace_include_filters_set()` | Include only the traces matching component list.           |
+|`mbed_trace_exclude_filters_get()` | Get pointer to the component inclusion filter list.        |
+|`mbed_trace_exclude_filters_set()` | Exclude the traces matching component list.                |
+
+The filter string is by default relatively short (24) and you can change its length by defining `DEFAULT_TRACE_FILTER_LENGTH`. If you use multiple components in the filter, separate them with a character that is not being used in the components name, such as comma (`,`). The matching is done simply using `strstr()` from  C standard libraries.
```

Comment:
This could be a bit less verbose while also clearer.
```
The filter list is a nul terminated string of comma (`,`) separated trace group names. The default maximum length of the string is 24 characters, including the terminating nul. This can be configured by defining the macro `DEFAULT_TRACE_FILTER_LENGTH` to the length you want.
```
Note: I can't remember if the terminating nul is actually included or not, nor whether spaces between groups in the list are allowed or not...

Reply:
I'll use your def and re-write this chapter a bit.